schema.ts

```diff
@@ -1117,6 +1130,12 @@ export interface IndexDefinition {
    */
   properties: IndexedProperty[];
 
+  /**
+   * A list of properties from within {@link ObjectSchemaDefinition.properties} 
```

Comment:
Maybe also note something like "Only up to 5 properties can be filterable, so these should be the properties most likely to be useful as filters."